api/v2alpha1/pipeline_spec.proto

```diff
@@ -852,6 +853,11 @@ message PipelineDeploymentConfig {
   }
   // Map from executor label to executor spec.
   map<string, ExecutorSpec> executors = 1;
+
+  // begin platform level / workflow level config
+
+  // duration in seconds after which the pipeline platform will do garbage collection
+  optional int32 completed_pipeline_ttl = 2;

```

Comment:
would suggest changing this type to `duration`. `Duration` explicitly represents time intervals, making the intent of the field more apparent